request_handler.py

```diff
@@ -0,0 +1,70 @@
+import importlib
+import methods
+import copy
+import methods
+from errors import FunctionError, TetherError
+from traceback import format_exc
+
+def execute(request):
+    """
+    Executing requests using the methods defined in the methods module.
+    
+    Request dict format:
+    {"func" : <function name that is in the methods module>,
+     "requests" : <list of dicts, functions and arguments to run in order. Not used if "func" is not equal to "tether">, 
+     "args" : <ordered arguments for func, including data input. For tether, these are the initial input arguments>,
+     "kwargs" : <key worded arguments for func>
+    }
+    
+    input: request (dict)
+    output: <tether function> OR method.func(*args, **kwargs)
+    """
+    
+    args = request.get("args",[])
+    kwargs = request.get("kwargs",{})
+        
+    func_name = request["func"]
+    if func_name == "tether":
+        try:
+            return tether(request["requests"], args, kwargs)
+        except FunctionError as e:
+            raise TetherError(request_info = e.info)
```

Comment:
not sure why you have a different kind of error here or why you want to return e.info

Reply:
i think if there is an error in a function in the pipeline of functions in a tether call, it should give an error that there was a problem in tether, and then the request_info would be the specific info to the function that threw an exception